packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -197,19 +237,36 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
                 return resolvedUrl;
             }
             const rateLimiter = new RateLimiter(this.driverPolicies.maxConcurrentOrdererRequests);
-            const ordererRestWrapper = await RouterliciousOrdererRestWrapper.load(
-                tenantId,
-                documentId,
-                this.tokenProvider,
+            const ordererRestWrapper = await PerformanceEvent.timedExecAsync(
                 logger2,
-                rateLimiter,
-                this.driverPolicies.enableRestLess,
-                resolvedUrl.endpoints.ordererUrl,
-            );
-            // The service responds with the current document session associated with the container.
-            const discoveredSession = await ordererRestWrapper.get<ISession>(
-                `/documents/${tenantId}/session/${documentId}`,
-            );
+                {
+                    eventName: "GetCreateSessionToken",
+                    docId: documentId,
+                    details: JSON.stringify({
+                        enableRestLess: this.driverPolicies.enableRestLess,
+                    }),
+                },
+                async () => {
+                    return RouterliciousOrdererRestWrapper.load(
```

Comment:
Can we just add a performance event around the token retrieval within .load() implementation?

Reply:
Yes made that earlier comment yesterday about that.  And what you suggested could be one option. I would still like to collect telemetry who (which path) requested token. Maybe added param on load.

Reply:
We have few load types and they may give us needed granularity for now.